gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord, Angle
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations contained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters
+    ----------
+    obs_table : `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos : `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    @property
+    def offset(self):
+        """Offset of observations relative to the 
+        target position (`astropy.coordinates.Angle`).
+        """
+        pnt_pos = SkyCoord(self.obs_table['RA_PNT'],
+                           self.obs_table['DEC_PNT'],
+                           unit='deg')
+
+        offset = pnt_pos.separation(self.target_pos)
+        
+        return offset
+    
+    def plot_zenith_distribution(self, ax=None, bins=None):
+        """
+        Construct the zenith distribution of the observations
+        
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None, optional.
+            The `~matplotlib.axes.Axes` object to be drawn on.
+            If None, uses the current `~matplotlib.axes.Axes`.
+        bins : integer
```

Comment:
I would suggest to put the following more accurate description of the `bins` option here and in the other plot method:
```python
"""
bins : integer or array_like, optional
    Binning specification, passed to `matplotlib.pyplot.hist`.
    By default, 30 bins from 0 deg to max zenith + 5 deg is used.
```
This will create a link from the Gammapy docs to the matplotlib docs here:
http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.hist